data/campaigns/The_Hammer_of_Thursagan/scenarios/08_Fear.cfg

```diff
@@ -185,6 +188,7 @@
     [/event]
 
     [event]
+        id=capture
```

Comment:
I think I'd prefer slightly more descriptive IDs, or at least ones that aren't identical to the event name.